lib/auxiliary/time_helper.rb

```diff
@@ -4,9 +4,11 @@
 module AutoHCK
   # Helper module
   module Helper
-    def time_to_seconds(time)
-      time.split(':').reverse.map.with_index { |a, i| a.to_i * (60**i) }
-          .reduce(:+)
+    def time_to_seconds(time_str)
+      days, time_part = time_str.split('.')
```

Comment:
```suggestion
      days, time_part = time_str.split('.', 2)
```

Looking at .NET's own parser:
https://github.com/dotnet/runtime/blob/v8.0.1/src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.TimeSpan.C.cs

There are the following possible formats:
- dd
- hh:mm
- dd.hh:mm
- hh:mm:ss
- dd.hh:mm:ss
- hh:mm:ss.fffffff
- dd.hh:mm:ss.fffffff

As shown here, `.` may appear before the second fraction too. We don't want to split it.

Also let's name `days` as `days_str` to make it consistent with `time_str` and to make it clear it is a string unlike `hours`, `minutes` and `seconds`.

Reply:
@akihikodaki I am ok with adding tests, just a question. Do we need to think about these formats:

- dd
- hh:mm
- dd.hh:mm
- hh:mm:ss.fffffff
- dd.hh:mm:ss.fffffff

All formats looks ok for MS, but I don't think HLK will provide something in its. Or at least ignore `second fraction`.

Reply:
We will never have the following according to the documentation https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings and source code https://github.com/dotnet/runtime/blob/v8.0.1/src/libraries/System.Private.CoreLib/src/System/Globalization/TimeSpanFormat.cs
- dd
- hh:mm
- dd.hh:mm

S second fraction is still possible. I think it's easy to convert to use a floating point number or just to ignore though (`to_i` will ignore the fraction).